### ISSUE TITLE
Portfolio Loading and returns no data

### ISSUE THREAD

**Issue by AKprisonER** [Cybourgeoisie/cryptofolio#7]
I can see Loading of circular icon and Data is not fetched from the CMC or cc

**Comment by AKprisonER** [comment]
when opened in incognito mode, sometimes loading my portfolio and sometimes always loading icon for each value can be seen. Is it browser problem or API request limit problem....In normal mode also not working. Both on my mobile n PC.

**Comment by Cybourgeoisie** [comment]
I caught this, too. It's driving me nuts. I'm going to add a cached version of the CoinMarketCap data as a fallback if the most recent data call fails or takes too long to load. I'll push an update shortly.

**Comment by Cybourgeoisie** [comment]
I put up a stop-gap - if CMC doesn't return with the API results within half a second, the call cancels and pulls a cached version of the CMC results. I'm going to refine it a bit more before committing the code, but the fix is live on the 10k.pizza site.

**Comment by Cybourgeoisie** [comment]
The stop-gap is effective - I'm going to add a cron job to automatically update the cache and store it on S3. This will be something that everyone can pull from, though there may be some CORS configuration issues for people running from localhost until all the kinks are worked out.

**Comment by AKprisonER** [comment]
Well,is this gonna be same one page ?

**Comment by Cybourgeoisie** [comment]
Good point. I can add a coin listing to this repository, but leave out the prices, which will be automatically updated when the CryptoCompare data comes in. I'll just need to regularly update the document as new coins are added to the CMC / CryptoCompare sites. That should resolve the problem without adding any new moving parts.

**Comment by AKprisonER** [comment]
well instead of adding coin each time, you can use ticker limit to get the coin list. I think currently there is 1500 coins are there it seems. So you can set ticker limit to 1600, so that even any new coins get added, it can automatically fetch it

**Comment by AKprisonER** [comment]
I switched from Android to IOS.
But 10k.pizza doesn't store my portfolio locally.
Is there any pblm with IOS browsers ?

**Comment by Cybourgeoisie** [comment]
That's weird - this is the first time I've heard of this issue. What version of iOS, which phone, and what browser? I'll see if I can get my hands on a device soon.
